fuzzing/README.md

```diff
@@ -0,0 +1,17 @@
+# suricata-fuzzing
+This project contains fuzz-targets for several interfaces and network protocols.  
+
+## Preparation
+To build the fuzz-targets the following requirement:
+* suricata
+* jansson 
+* libnet 
+* libyaml
+
+
+## build and test
+Steps for build and test the fuzz-targets using some scripts :
+* create fuzz-envirorment with **docker**
+* run **build.sh** to compile *suricata*
+* run **patch.sh** to modify suricata.o
```

Comment:
I think that we should have only one version of Suricata, where fuzzing can be enabled at configure time with an option
Something like https://github.com/OISF/suricata/pull/3837/files#diff-67e997bcfdac55191033d57a16d1408aR2378

Reply:
right, we have to implement this in the main makefile